releasenotes/notes/ray-core-integration-91f88d121822166d.yaml

```diff
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    ray: This introduces a Ray core integration that traces Ray jobs, remote
+    tasks, and actor method calls. Supported for Ray >= 2.46.0.
+
+    To enable tracing, start the Ray head with
+    `--tracing-startup-hook=ddtrace.contrib.internal.ray.hook:setup_tracing`,
```

Comment:
We should expose the `setup_tracing` method in `ddtrace.contrib.ray.__init__`. This will make this operation public. We should avoid documenting internal apis in our docs.

```suggestion
    `--tracing-startup-hook=ddtrace.contrib.ray:setup_tracing`,
```